src/Service/Context/PageContext.php

```diff
@@ -67,7 +74,6 @@ class PageContext {
     'account_name' => 'account_id',
```

Comment:
If acquia-lift-decision-api and lift3 use the machine identifier site_id and account_id, we should follow those conventions in the Drupal module that utilizes both of those services. This is what was agreed on with Lift Web as well and is also used in the Lift Web REST API http://apidoc.dev.lift.acquia.com/